packages/arui-scripts/package.json

```diff
@@ -75,58 +73,59 @@
         "postcss-omit-import-tilde": "^1.0.1",
         "postcss-strip-units": "^2.0.1",
         "postcss-url": "7.3.1",
-        "react-dev-utils": "6.1.1",
-        "react-hot-loader": "4.12.19",
+        "react-dev-utils": "11.0.4",
+        "react-refresh": "0.10.0",
+        "react-refresh-typescript": "2.0.2",
         "reload-server-webpack-plugin": "1.0.1",
         "rimraf": "^2.6.2",
+        "run-script-webpack-plugin": "0.0.11",
         "shelljs": "0.8.3",
-        "start-server-webpack-plugin": "^2.2.1",
         "strip-ansi": "^6.0.0",
         "style-loader": "1.1.3",
         "svg-url-loader": "^6.0.0",
         "tar": "4.4.8",
-        "terser-webpack-plugin": "1.4.5",
+        "terser-webpack-plugin": "5.2.4",
         "ts-jest": "26.5.5",
-        "ts-loader": "6.2.1",
-        "ts-node": "^8.10.2",
-        "tsconfig-paths-webpack-plugin": "^3.2.0",
-        "url-loader": "1.1.2",
-        "webpack": "4.41.5",
-        "webpack-bundle-analyzer": "^3.8.0",
-        "webpack-dev-server": "3.10.1",
-        "webpack-manifest-plugin": "2.2.0",
-        "webpack-node-externals": "^1.6.0"
+        "ts-loader": "9.2.3",
+        "ts-node": "9.1.1",
+        "tsconfig-paths-webpack-plugin": "3.5.1",
+        "url-loader": "4.1.1",
```

Comment:
https://v4.webpack.js.org/loaders/url-loader/

DEPREACTED for v5: please consider migrating to [asset modules](https://webpack.js.org/guides/asset-modules/).